Lib/glyphsLib/classes.py

```diff
@@ -3357,14 +3357,12 @@ def windowsLinkedToStyle(self):
 
     @property
     def fontName(self):
-        value = (
+        return (
             self.properties.get("postscriptFontName")
             or self.customParameters["postscriptFontName"]
```

Comment:
I would convert all custom parameters on loading the file. That makes all the accessor code much simpler.

Reply:
> I would convert all custom parameters on loading the file.

I don't disagree it'd be desirable, but upconverting glyphs v2 files to a v3 internal representation upon reading (and down-converting to v2 if requested upon writing) would involve some major refactoring.